kani-driver/src/call_goto_instrument.rs

```diff
@@ -57,7 +57,8 @@ impl KaniSession {
             }
 
             let c_demangled = alter_extension(output, "demangled.c");
-            let symtab = project.get_harness_artifact(&harness, ArtifactType::SymTab).unwrap();
+            let symtab =
```

Comment:
Can you update the variable name (`symtab`)?

Reply:
fixed